### PR TITLE
fix(#patch); stargate; add Base config

### PR DESCRIPTION
**Pull request by dhruv-chauhan**
Deployments,

- stargate-ethereum: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-mainnet
- stargate-arbitrum: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-arbitrum
- stargate-avalanche: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-avalanche
- stargate-bsc: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-bsc
- stargate-fantom: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-fantom
- stargate-optimism: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-optimism
- stargate-polygon: https://okgraph.xyz/?q=dhruv-chauhan%2Fstargate-polygon

### CODE REVIEW

File: schema-bridge.graphql [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # Subgraph Schema: Bridge
-# Version: 1.2.0
+# Version: 1.2.1
```
Comment (dmelotik) [inline schema-bridge.graphql:2]: Don't need to increment version for this

File: subgraphs/stargate/src/sdk/protocols/bridge/chainIds.ts [hunk 2]
```diff
@@ -40,3 +41,7 @@ setChainID(11, Network.OPTIMISM);
 setChainID(111, Network.OPTIMISM);
 setChainID(12, Network.FANTOM);
 setChainID(112, Network.FANTOM);
+setChainID(51, Network.METIS);
+setChainID(151, Network.METIS);
```
Comment (dmelotik) [inline subgraphs/stargate/src/sdk/protocols/bridge/chainIds.ts:45]: Where does this come from? I must be missing something https://chainlist.org/chain/1088

File: subgraphs/stargate/protocols/stargate/config/deployments/stargate-base/configurations.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "deployment": "STARGATE_BASE",
+  "network": "base",
```
Comment (dmelotik) [inline subgraphs/stargate/protocols/stargate/config/deployments/stargate-base/configurations.json:3]: I think we are getting a build error bc this isn't the proper network name for the graph node

Comment (dmelotik) [reply subgraphs/stargate/protocols/stargate/config/deployments/stargate-base/configurations.json:3]: You should be able to find it in the graph node release notes or source code

Comment (dhruv-chauhan) [reply subgraphs/stargate/protocols/stargate/config/deployments/stargate-base/configurations.json:3]: problem was with the lpStaking dataSource naming, which is used in imports; fixed it.

**Review by dmelotik — APPROVED** [review]
lgtm! Feel free to deploy @dhruv-chauhan

**Pull request merged**